tests/utils/shippable/shippable.sh

```diff
@@ -75,8 +75,8 @@ set -ux
 
 pip install setuptools==44.1.0
 
-if [ -n "${ANSIBLE_BASE_REV:-}" ]; then
-    pip install "ansible~=${ANSIBLE_BASE_REV}"
+if [ -n "${A_REV:-}" ]; then
+    pip install "ansible~=${A_REV}"
```

Comment:
Unfortunately this needs to be a bit more complex since you probably want to use `ansible` for 2.9 and `ansible-base` for 2.10 and later. Also, since you want to test a pre-release, `~=` doesn't work, even with `--pre`, so you'll need to pick a specific version. Either that or specify a branch name instead of a version number.

Here's one possibility using version numbers:

```suggestion
if [[ "${A_REV:-}" == "2.9" ]]; then
    install_package="ansible~=2.9"
elif [[ "${A_REV:-}" == 2.10 ]]; then
    install_package="ansible-base==2.10.0b1"
fi

if [ -n "${install_package:-}" ]; then
    pip install "${install_package}"
```

If you wanted to specify a branch name you could do something like `A_BR=2.10` and have the script use `"stable-${A_BR}"` instead of `"devel"` if it is set (instead of using `A_REV`).